pkg/cli/clients/clients.go

```diff
@@ -164,10 +164,10 @@ type ApplicationsManagementClient interface {
 	ListEnvironmentsAll(ctx context.Context) ([]corerp.EnvironmentResource, error)
 	GetEnvDetails(ctx context.Context, envName string) (corerp.EnvironmentResource, error)
 	DeleteEnv(ctx context.Context, envName string) (bool, error)
-	CreateUCPGroup(ctx context.Context, planeType string, planeName string, resourceGroupName string, resourceGroup ucp_v20231001preview.ResourceGroupResource) error
```

Comment:
This is one of the more viral changes. Previous the resource group API documented that you could create a resource group in any plane. This is clearly false, it's just the `radius` plane. I fixed that mistake in the typespec which resulted in a change everywhere the resource group client is used.